saturnbase-rstudio/Dockerfile

```
@@ -0,0 +1,186 @@
+        procps \
+        rsync \
+        sudo \
+        screen \
+        wget \
+     > /dev/null && \
+    apt-get -qq purge && \
+    apt-get -qq clean && \
+    rm -rf /var/lib/apt/lists/* && \
+    mkdir -p /run/sshd && \
+    chmod 755 /run/sshd && \
+    echo "en_US.UTF-8 UTF-8" > /etc/locale.gen && \
+    locale-gen
+
+ENV PATH /opt/conda/bin:$PATH
+
+# Based on https://github.com/ContinuumIO/docker-images/blob/4d7798c0ea2463d9c4057d8eaee876102eecbf86/miniconda3/debian/Dockerfile
+ARG CONDA_VERSION=py39_4.10.3
+RUN set -x && \
+    MINICONDA_URL="https://repo.anaconda.com/miniconda/Miniconda3-${CONDA_VERSION}-Linux-x86_64.sh"; \
+    SHA256SUM="1ea2f885b4dbc3098662845560bc64271eb17085387a70c2ba3f29fff6f8d52f"; \
+    wget "${MINICONDA_URL}" -O miniconda.sh -q && \
+    echo "${SHA256SUM} miniconda.sh" > shasum && \
+    if [ "${CONDA_VERSION}" != "latest" ]; then sha256sum --check --status shasum; fi && \
+    mkdir -p /opt && \
+    sh miniconda.sh -b -p /opt/conda && \
+    rm miniconda.sh shasum && \
+    ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh && \
+    echo ". /opt/conda/etc/profile.d/conda.sh" >> ~/.bashrc && \
+    echo "conda activate base" >> ~/.bashrc && \
+    find /opt/conda/ -follow -type f -name '*.a' -delete && \
+    find /opt/conda/ -follow -type f -name '*.js.map' -delete && \
+    /opt/conda/bin/conda clean -afy && \
+    ln -s /opt/conda /srv/conda && \
+    chown -R 1000:1000 /opt/conda && \
+    chown -R 1000:1000 /srv
+
+ENV LC_ALL=en_US.UTF-8
+ENV LANG=en_US.UTF-8
+ENV LANGUAGE=en_US.UTF-8
+ENV SHELL=/bin/bash
+ENV NB_USER=jovyan
+ENV NB_UID=1000
+ENV USER=${NB_USER}
+ENV HOME=/home/${NB_USER}
+
+RUN adduser --disabled-password \
+    --gecos "Default user" \
+    --uid ${NB_UID} \
+    ${NB_USER} && \
+    echo "$NB_USER ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers.d/notebook
+COPY profile /etc/profile
+
+USER ${NB_USER}
+
+COPY setup-conda.bash /tmp/setup-conda.bash
+COPY environment.yml /tmp/environment.yml
+RUN bash /tmp/setup-conda.bash && \
+    echo '' > ${CONDA_DIR}/conda-meta/history && \
+    ${CONDA_BIN}/conda config --system --add channels conda-forge && \
+    ${CONDA_BIN}/conda config --system --set auto_update_conda false && \
+    ${CONDA_BIN}/conda config --system --set show_channel_urls true
+ENV NPM_DIR ${APP_BASE}/npm
+ENV NB_PYTHON_PREFIX ${CONDA_DIR}/envs/saturn
+ENV PATH ${NB_PYTHON_PREFIX}/bin:${CONDA_BIN}:${NPM_DIR}/bin:${HOME}/.local/bin:${PATH}
+WORKDIR ${HOME}
+
+## Install R
+
+RUN sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-key '95C0FAF38DB3CCAD0C080A7BDC78B2DDEABC47B7' \
+    && sudo apt-get update \
+    && sudo apt-get install -y \
+    --no-install-recommends \
+        software-properties-common
+
+RUN sudo su -c \
+    "echo 'deb http://cloud.r-project.org/bin/linux/debian bullseye-cran40/' >>/etc/apt/sources.list"
+
+RUN sudo apt update && sudo apt-get install -y -t bullseye-cran40 \
+    r-base \
+    r-base-core \
+    r-base-dev 
```

```suggestion
RUN sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-key '95C0FAF38DB3CCAD0C080A7BDC78B2DDEABC47B7' && \
    sudo su -c \
    "echo 'deb http://cloud.r-project.org/bin/linux/debian bullseye-cran40/' >>/etc/apt/sources.list" && \
    sudo apt update && \
    sudo apt-get install -y -t bullseye-cran40 \
        r-base \
        r-base-core \
        r-base-dev 
```

And move `software-properties-common` into the main `apt` block.